.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
   "typescript.tsdk": "node_modules/typescript/lib"
-}
+}
```

Comment:
We can probably revert this change.

This is non-blocking feedback.

Reply:
🤣 Yes. I will not pere the PR (hence the title). If we think this is an approach we want to go with I would create a new clean PR.